assembl/static2/js/app/components/administration/survey/questionSection.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { gql, withApollo } from 'react-apollo';
+import { Link } from 'react-router';
+import { I18n } from 'react-redux-i18n';
+import { Row, Col } from 'react-bootstrap';
+
+import VideoForm from './videoForm';
+import QuestionsForm from './questionsForm';
+import SectionTitle from '../sectionTitle';
+import { getDiscussionSlug } from '../../../utils/globalFunctions';
+
+const GetThematics = gql`
+{
+  thematics(identifier:"survey") {
+    id,
+    titleEntries {
+      localeCode,
+      value
+    },
+    imgUrl,
+    video {
+      title,
+      description,
+      htmlCode
+    },
+    questions {
+      titleEntries {
+        localeCode,
+        value
+      }
+    }
+  }
+}
+`;
+
+const QuestionSection = ({ client, i18n, selectedLocale, thematicId }) => {
+  const thematicsData = client.readQuery({ query: GetThematics });
```

Comment:
we should probably create an util to get the thematics from apollo's cache and fallback to an empty list of thematics